cdk/domino_cdk/provisioners/eks/__init__.py

```diff
@@ -41,8 +41,10 @@ def __init__(
         )
 
         cdk.CfnOutput(parent, "eks_cluster_name", value=self.cluster.cluster_name)
+
+        region = cdk.Stack.of(self.scope).region
```

Comment:
What is this doing differently?

Reply:
mypy complains, but ultimately I think this is basically casting the "reusable" `Construct` into ` Stack` that has the actual region attribute. Question is whether or not we should be pedantically correct, I guess.